### PR TITLE
Consolidate getting started instructions into dev/README.md

### PR DESCRIPTION
**Pull request by khmoran**
To honor Spencer's sacrifice of being our guinea pig and getting the stack up and running, a few updates to our documentation to help future brave souls.

@SpencerMarks-IDHA: These are just my notes from our convo last week but definitely feel free to add in more!

### CODE REVIEW

File: dev/README.md [hunk 1]
```diff
@@ -1,27 +1,72 @@
...
+the curator UI to manage case report data sources and view or update case data, with more functionality coming soon.
 
-```
+## Getting up and running
+
+### Prerequisites
+
+#### Software
+
+- [Homebrew](https://docs.brew.sh/Installation): To facilitate installing the below.
```
Comment (smarks) [inline dev/README.md:20]: what about any unfortunates who are using a non-mac platform?

Comment (khmoran) [reply dev/README.md:20]: I sympathize with these unfortunates, being one of them myself. That link has a polite nod to "alternative installs" for linux (and windows 10 subsystem for linux, whatever that means)

File: dev/README.md [hunk 2]
```diff
@@ -1,27 +1,72 @@
...
+- [MongoDB](https://docs.mongodb.com/manual/administration/install-community/): Make sure, after install, you can run
```
Comment (attwad) [inline dev/README.md:21]: nit. in practice we're using the docker image of mongo so you don't need to install mongodb (not a *prerequisite*), but the CLI tools are nice to have (mongoimport and others

Comment (khmoran) [reply dev/README.md:21]: sadly they are needed for setup_db script :(

**Pull request merged**